LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2023] [RustaceansKenya]
```

Comment:
Remove the brackets as they are not part of the Copyright. It should be
Copyright 2023 RustaceansKenya

Reply:
Forgot we are in 2024